src/pages/journal/components/candidate-link/__tests__/candidate-link.spec.ts

```diff
@@ -43,12 +43,13 @@ describe('CandidateLinkComponent', () => {
       expect(component).toBeDefined();
     });
 
-    it('should call the push function of navController and pass the right slotId', () => {
-      component.navigateToCandidateDetails();
+    it('should call the create function of navController and pass the right slotId', () => {
```

Comment:
The name of this test still says `navController`.

Reply:
changed